proto/desmos/profiles/v1beta1/query.proto

```diff
@@ -66,9 +69,11 @@ service Query {
         "/desmos/profiles/v1beta1/chain-links/{user}/{chain_name}/{target}";
   }
 
-  // UserApplicationLinks queries application links for the given user
-  rpc UserApplicationLinks(QueryUserApplicationLinksRequest)
-      returns (QueryUserApplicationLinksResponse) {
+  // ApplicationLinks queries all application links
+  // It queries the applications links for the given user if the user address is
+  // provided
```

Comment:
```suggestion
  // ApplicationLinks queries the applications links associated to the given user, if provided.
  // Otherwise, it queries all the application links stored.
```